rmgpy/tools/loader.py

```diff
@@ -36,10 +36,8 @@
 import os.path
```

Comment:
Can you add a constant species to the liquid sensitivity test to improve our coverage?